Gopkg.toml

```diff
@@ -43,11 +43,11 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.11.2"
+  version = "kubernetes-1.13.1"
```

Comment:
For such PR, we prefer two commits, one commit is the update for Gopkg.toml, another commit is for `dep ensure`, the auto-generated code, this can make the PR more easy for review.

Reply:
ok ,I might reach you for multiple commit push, I knew gerrit but not for github :)

Reply:
@jichenjc for two commits, you can do as follows:
1) Update `Gopkg.toml`, and commit with `git commit -a`, Do not push.
2) Run `dep ensure` and it will update vendor and generate some code, commit this again with `git commit -a`.
3) `git push`, this will make your PR has two commits ;-)

Reply:
ok, similar to gerrit review, I will see how I can handle it, thanks :)